### PR TITLE
Flamegraph: Toggle between 1 and 8 parallel mget requests

### PR DESCRIPTION
**Pull request by rockdaboot**
This is already currently deployed.
I would like to merge it into main so we don't lose it with the next deployment, even if it is just temporary.

### CODE REVIEW

File: src/plugins/profiling/server/routes/search_flamechart.ts [hunk 1]
```diff
@@ -170,12 +172,13 @@ async function queryFlameGraph(
   }
 
   // profiling-stacktraces is configured with 16 shards
-  const nQueries = 8;
+  const nQueries = parallel ? 8 : 1;
+  parallel = !parallel;
```
Comment (inge4pres) [inline src/plugins/profiling/server/routes/search_flamechart.ts:176]: I am not sure I understand what we're doing here: isn't `parallel` going to be always `true` when we execute this?

**Review by inge4pres — COMMENTED** [review]
Sorry, maybe my brain is fried but I am not getting what we are trying to achieve :|

File: src/plugins/profiling/server/routes/search_flamechart.ts (hunk 1 above)
Comment (rockdaboot) [reply src/plugins/profiling/server/routes/search_flamechart.ts:176]: No, this is how you toggle a boolean value :-)

So this is already been deployed for testing. We want to keep it for a while until we are absolutely sure that parallel requests are good or not.

**Review by inge4pres — APPROVED** [review]
Approving this as a temporary workaround, **this PR will be reverted in the future**

**Pull request merged**